microsoft/PowerToys
# [FZ Editor] Spacing settings in layout flyouts

## Summary of the Pull Request

Added spacing and sensitivity radius properties to layout models. 
Added info into `zones-settings.json` to keep different settings per custom layout.

```
{
    "type": "grid",
    "info": {
        ...
        "show-spacing": true,
        "spacing": 16,
        "sensitivity-radius": 20
    }
}
```
```
{
    "type": "canvas",
    "info": {
        ...
        "sensitivity-radius": 20
    }
}
```
## PR Checklist
* [x] Applies to #8715

## Validation Steps Performed

Set different spacing and sensitivity radius settings to layouts and verify that they were saved and applied properly.
